test/specs/anInitialSuite.e2e.ts

```diff
@@ -96,34 +88,27 @@ describe('An Initial Suite', async () => {
   step('Set up the testing environment', async () => {
     testSetup = new TestSetup('AnInitialSuite', false);
     // Don't call testSetup.setUp() b/c we don't need to authorize a scratch org,
-    // just call setUpTestingEnvironment() and createProject().
+    // just call setUpTestingEnvironment() and createInitialProject().
     await testSetup.setUpTestingEnvironment();
-    await testSetup.createProject('Standard');
+    await testSetup.createInitialProject('Standard');
     await utilities.reloadAndEnableExtensions();
   });
 
   step('Verify our extensions are loaded after creating an SFDX project', async () => {
-    utilities.verifyAllExtensionsAreRunning();
+    await utilities.verifyExtensionsAreRunning(utilities.getExtensionsToVerifyActive());
```

Comment:
Another new utility that takes a set of extension Ids to verify if they are present.